packages/orchestration/src/orchestration-client.ts

```diff
@@ -85,3 +140,13 @@ export function constructCompletionPostRequest(
     })
   };
 }
+
+function configureLLM(llm: LlmModuleConfig, stream: boolean): LlmModuleConfig {
+  if (stream) {
+    llm.model_params = {
+      ...llm.model_params,
+      stream_options: { include_usage: true }
```

Comment:
(question) is this API harmonized? Maybe test it with e.g. Gemini?

Reply:
I was raising the same question, but at least also for Gemini it works as intended.